apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyKeyInfoHarness } from '@skyux/indicators/testing';
+
+import { KeyInfoDemoComponent } from './key-info-demo.component';
+import { KeyInfoDemoModule } from './key-info-demo.module';
+
+describe('Basic key info', () => {
+  async function setupTest(options?: { layout?: 'vertical' | 'horizontal' }) {
+    const fixture = TestBed.createComponent(KeyInfoDemoComponent);
+
+    if (options?.layout !== undefined) {
+      fixture.componentInstance.layout = options.layout;
+    }
+
+    const loader = TestbedHarnessEnvironment.loader(fixture);
+
+    const keyInfoHarness = await loader.getHarness(
+      SkyKeyInfoHarness.with({ dataSkyId: 'key-info-demo' })
+    );
+
+    return { keyInfoHarness, fixture };
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [KeyInfoDemoModule],
+    });
+  });
+
+  it('should display a vertical key info', async () => {
+    const { keyInfoHarness, fixture } = await setupTest({ layout: 'vertical' });
+    fixture.detectChanges();
```

Comment:
Are these `fixture.detectChanges` needed? Just verifying as I know we were seeing that the harness loader seemed to be doing that by default at one point.

Reply:
they apparently aren't needed. removed